simtools/applications/simulate_showers_for_trigger_rates.py

```diff
@@ -107,9 +120,29 @@ def _parse(label=None, description=None):
         required=True,
     )
     config.parser.add_argument("--nruns", help="Number of runs", type=int, default=100)
+    config.parser.add_argument(
+        "--run_number", help="Run number of the starting run", type=int, default=1
+    )
     config.parser.add_argument("--nevents", help="Number of events/run", type=int, default=100000)
     config.parser.add_argument("--zenith", help="Zenith angle in deg", type=float, default=20)
     config.parser.add_argument("--azimuth", help="Azimuth angle in deg", type=float, default=0)
+    config.parser.add_argument("--view_cone", help="View cone in deg", type=float, default=10)
+    config.parser.add_argument(
+        "--scatter_x", help="Scatter distance (X axis) in m", type=float, default=1500
+    )
+    config.parser.add_argument(
+        "--scatter_y", help="Scatter radius  (Y axis) in m", type=float, default=0
```

Comment:
Technical this is not a radius. If y is 0, x is a radius. If y > 0., then the scatter area is a square. I suggest to only allow the user to set x, as I don't know of any use case for CTA where we do not want to simulate the events on a circular area.

Reply:
True, this is not a radius. I adjusted the text now. I think there is no specific use case for scatter Y, but as the array might not be symmetric either, it might make a difference to spread events on X or Y (of course, depending on the array layout). I can remove it for the moment, as we do not need it now.